lib/ecto/multi.ex

```diff
@@ -659,14 +684,26 @@ defmodule Ecto.Multi do
     end
   end
 
+  defp apply_operation({name, {:inspect, opts}}, _repo, _wrap_, _return, {acc, names}) do
+    opts = Keyword.merge([label: Atom.to_string(name)], opts)
+
+    if opts[:only] do
+      acc |> Map.take(opts[:only]) |> IO.inspect(opts)
```

Comment:
For convenience, please make sure this also works: `inspect(only: :foo)`:

```suggestion
      acc |> Map.take(List.wrap(opts[:only])) |> IO.inspect(opts)
```

Will also need a test. :)